test/test_dtr.py

```diff
@@ -730,6 +730,82 @@ def test_exception(self, lhs, rhs, expected):
             lhs.is_intersection(rhs)
 
 
+class TestDateTimeRange_subtract:
+    @pytest.mark.parametrize(
+        ["lhs", "rhs", "expected"],
+        [
```

Comment:
Maybe add a test case for when the `lhs` and `rhs` do not overlap?

Reply:
Addressed. See third test case.